components/com_apps/models/category.php

```diff
@@ -256,6 +256,9 @@ private function loadExtensionIDs($db, $catid)
 	
 	public function getExtensions()
 	{
+		
+		return array();
```

Comment:
This is WIP. What's left is the extensions from category and
parent-childing the left sidebar.

don't think json_decode throws and in case of invalid json we already show
an error.

On Sat, Jan 17, 2015 at 5:04 PM, beat <notifications@github.com> wrote:

> In components/com_apps/models/category.php
> <https://github.com/joomla-extensions/install-from-web-server/pull/4#discussion-diff-23124663>
> :
>
> > @@ -256,6 +256,9 @@ private function loadExtensionIDs($db, $catid)
> >  	
> >  	public function getExtensions()
> >  	{
> > +		
> > +		return array();
>
> Due to return-line added at begin of function, is the now unreachable code
> below left on purpose or is this WIP ?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla-extensions/install-from-web-server/pull/4/files#r23124663>
> .
>



-- 
Thanks!
Ashwin Date
Tekdi Web Solutions - tekdi.net | techjoomla.com
Twitter - @coolbung | @techjoomla | @tekdinet

If it isn't broken, don't fix it!